source/common/access_log/access_log_formatter.cc

```diff
@@ -221,6 +223,18 @@ RequestInfoFormatter::RequestInfoFormatter(const std::string& field_name) {
     field_extractor_ = [](const RequestInfo::RequestInfo& request_info) {
       return AccessLogFormatUtils::durationToString(request_info.firstUpstreamRxByteReceived());
     };
+  } else if (field_name == "TIME_TO_LAST_BYTE_DURATION") {
```

Comment:
Shouldn't "time to last byte" report the time from the start of the downstream request until the last tx byte? (Which is just DURATION.) This is measuring how long the response takes to transit through Envoy, so more like RESPONSE_TRANSIT_DURATION or RESPONSE_TX_DURATION or something?

Reply:
I like `RESPONSE_TX_DURATION` — I think it's the closes one to "the time between the first upstream byte and the last downstream byte sent". 

cc: @derekargueta